services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+from datetime import datetime
+from xmlrpc.client import DateTime
+
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+    movie_show_time: DateTime,
+    movie_id: int,
+    cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=Movie.objects.get(id=movie_id),
+        cinema_hall=CinemaHall.objects.get(id=cinema_hall_id)
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> MovieSession:
+    if session_date:
+        return MovieSession.objects.filter(
+            show_time__date=datetime.strptime(session_date, "%Y-%m-%d")
+        )
+    return MovieSession.objects.all()
+
+
+def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
+    return MovieSession.objects.get(id=movie_session_id)
+
+
+def update_movie_session(
+        session_id: int,
+        show_time: DateTime = None,
+        movie_id: int = None,
+        cinema_hall_id: int = None

```

Comment:
Add Optional to annotations where the default arguments are `None`

Datetime (from xmlrpc.client import DateTime) is a wrong annotation for `show_time`, you can use just datetime (from standard datetime module)